### PR TITLE
Modified kRnel to compile and run with rust 0.9

### PR DESCRIPTION
**Pull request by dkhenry**
Modified the source to compile with rust 0.9. 

As part of this I removed all the #[author = "..."] and #[license = "..."] attributes since they are not supported in 0.9 I don't know I am going to submit another pull request that add in the traditional license block and a license file.

### CODE REVIEW

File: src/kernel/main.rs [hunk 1]
```diff
@@ -1,10 +1,17 @@
...
-       author = "Arcterus",
-       license = "MPL v2.0")];
+/*
+ * Copyright (c) 2014 Arcterus@mail.com
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */ 
+#[crate_id = "kRnel#0.0.1"];
```
Comment (Arcterus) [inline src/kernel/main.rs]: Can you please change `#[crate_id = "kRnel#0.0.1]` to:

```
#[crate_id(name = "kRnel",
           vers = "0.0.1",
           author = "Arcterus",
           license = "MPL v2.0")];
```

File: src/drivers/io/console/arch/x86_64/console.rs [hunk 2]
```diff
@@ -1,6 +1,10 @@
-#[author = "Arcterus"];
-#[license = "MPL v2.0"];
-
+/*
+ * Copyright (c) 2014 Arcterus@mail.com
```
Comment (Arcterus) [inline src/drivers/io/console/arch/x86_64/console.rs]: Also, would you mind changing `Arcterus@mail.com` to just `Arcterus`?

File: src/kernel/main.rs [hunk 3]
```diff
@@ -30,7 +37,7 @@ pub mod error;
 #[start]
 pub fn main() {
 	console::clear_screen();
-	console::print("iiiiiiiiiiiiiiiiiiiiiiiiiii\niiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiii\x08\x08\x08\x08\x08test");
+	console::print("iiiiiiiiiiiiiiiiiiiiiiiiiii\niiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiii\x08\x08\x08\x08\x08test");
```
Comment (Arcterus) [inline src/kernel/main.rs]: There seems to be some added space after the function call.  Can you remove that too?

**Comment by dkhenry** [comment]
Should be all updated now

**Comment by Arcterus** [comment]
Thanks. :+1:

**Pull request merged**